draft-ietf-anima-brski-prm.md

```diff
@@ -558,65 +568,69 @@ These endpoints accommodate for the signature-wrapped objects used by BRSKI-PRM
 
 * Domain registrar EE certificate: certificate of the domain registrar to be provided to the pledge.
 * Registrar-Agent EE certificate and corresponding private key: own operational key pair to sign agent-signed-data.
-* Serial number(s): product serial number(s) of pledge(s) to be bootstrapped for discovery.
+* Serial number(s): product-serial-number(s) of pledge(s) to be bootstrapped; used for discovery.
 
 Further, the Registrar-Agent SHOULD have synchronized time.
 
-Finally, the Registrar-Agent MAY possess the IDevID (root or issuing) CA certificate of the pledge vendor/manufacturer to validate the IDevID certificate on returned PVR or in case of TLS usage for pledge communication.
+Finally, the Registrar-Agent MAY possess the IDevID (root or issuing) CA certificate of the pledge vendor/manufacturer to validate the IDevID certificate on returned PVR or in case of optional TLS usage for pledge communication (see {{pledgehttps}}).
 The distribution of IDevID CA certificates to the Registrar-Agent is out of scope of this document and may be done by a manual configuration.
 
 
 ### Discovery of the Registrar {#discovery_uc2_reg}
 
-As a Registrar-Agent acts as representative of the domain registrar towards the pledge or may even be collocated with the domain registrar, a separate discovery of the registrar is likely not needed as Registrar-Agent and registrar are domain components and have a trust relation.
-Moreover, other communication (not part of this document) between the Registrar-Agent and the registrar is assumed, e.g., to exchange information about product-serial-number(s) of pledges to be discovered as outlined in {{arch_nomadic}}.
-Moreover, as the standard discovery described in {{Section 4 of !RFC8995}} and the {{Appendix A.2 of !RFC8995}} does not support  of registrars with an enhanced feature set (like the support of BRSKI-PRM), this standard discovery is not applicable.
+While the Registrar-Agent requires the IP address of the domain registrar to initiate a TLS session, a separate discovery of the registrar is likely not needed and a configuration of the domain registrar IP address or hostname is assumed.
+Registrar-Agent and registrar are domain components that already have a trust relation, as a Registrar-Agent acts as representative of the domain registrar towards the pledge or may even be collocated with the domain registrar.
+Further, other communication (not part of this document) between the Registrar-Agent and the registrar is assumed, e.g., to exchange information about product-serial-number(s) of pledges to be discovered as outlined in {{arch_nomadic}}.
+
+Moreover, the standard discovery described in {{Section 4 of !RFC8995}} and the {{Appendix A.2 of !RFC8995}} does not support identification of registrars with an enhanced feature set (like the support of BRSKI-PRM), and hence this standard discovery is not applicable.
 
```

Comment:
We may provide here a reference to the ongoing discovery work, which exactly targets this feature discovery.  [BRSKI Discovery ](https://datatracker.ietf.org/doc/draft-ietf-anima-brski-discovery/)

Reply:
Sounds like you mainly mean to update the already existing `{{I-D.eckert-anima-brski-discovery}}` references, no?

Reply:
Uh, yes, this reference needs to be updated. Aehm I did not expand the part. So the reference is already there.

Reply:
Done.